test/utils/test_zenith_angle.py

```diff
@@ -0,0 +1,54 @@
+# ignore_header_test
+
+# climt/LICENSE
```

Comment:
Just checking if we should use the Apache license here? Or is this also third party code?

Reply:
like the zenith angle implementation, this code is from https://github.com/ai2cm/fv3net/tree/master.